src/app/public/src/modules/wrapper/wrapper.component.ts

```diff
@@ -94,6 +100,11 @@ export class StacheWrapperComponent implements OnInit, AfterContentInit, OnDestr
     );
   }
 
+  private checkEditButtonUrl(): boolean {
+    const url = _get(this.config, 'skyux.appSettings.stache.editButton.url');
```

Comment:
What was @Blackbaud-BrandonJones saying about changing `_get` to something more explanatory? Something like `lodashGet`?

Reply:
This was the suggestion. Prefixing with the underscore like the lodash package. Since we are just importing one function, I added it to the name instead. `_.get` = `_get`

Reply:
👍